.travis.yml

```diff
@@ -26,6 +23,15 @@ jobs:
       env: &Debug
         - TRAVIS_CUBERITE_BUILD_TYPE=Debug
 
+    - name: "Clang 6.0 - Debug"
```

Comment:
Is that compiler version accurate? If so, it's too old.

Reply:
ageist
It works, doesn't it?

Reply:
Well our `COMPILING.md` file says it's not a supported version. Why not just used gcc which seems to have the same version as the amd64 workers.

Reply:
From what I've seen clang tends to give more warnings, and we need them on these exotic arches xD

Reply:
You haven't done anything about this yet.

Reply:
What is your suggestion?

Reply:
>Why not just used gcc which seems to have the same version as the amd64 workers.

Reply:
In response I suggested that we might need the warnings Clang provides, especially when it highlights some assumption we make that might not hold true on ARM.

Is the compiler version a blocker for you? I think arm server builds are more of a "nice to know it works" instead of "we support this" so building with Clang 6 isn't a problem.